File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/Buttons_ViewModel.cpp

#include "Buttons_ViewModel.h"
#include "Buttons_ViewModel.h"
#include <string>

namespace widgetassertions
{
  std::string Buttons_ViewModel::getOkButtonText( ) 
  {
    return this->OkButtonText;
  }
  bool Buttons_ViewModel::getIsOkButtonEnabled( ) 
  {
    return this->isOkButtonEnabled;
  }
  bool Buttons_ViewModel::getIsOkButtonVisible( ) 
  {
    return this->isOkButtonVisible;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/Buttons_ViewModel.cs
/// <filename>
///     Buttons_ViewModel.cs
/// </filename>
namespace widgetassertions
{
    public class Buttons_ViewModel
    {
        private string OkButtonText;

        public virtual string getOkButtonText()
        {
            return this.OkButtonText;
        }

        private bool isOkButtonEnabled;

        public virtual bool getIsOkButtonEnabled()
        {
            return this.isOkButtonEnabled;
        }

        private bool isOkButtonVisible;

        public virtual bool getIsOkButtonVisible()
        {
            return this.isOkButtonVisible;
        }
    }
}
